packages/core/rollup.config.js

```diff
@@ -0,0 +1,43 @@
+import babel from '@rollup/plugin-babel';
+import commonjs from '@rollup/plugin-commonjs';
+import json from '@rollup/plugin-json';
+import resolve from '@rollup/plugin-node-resolve';
+import replace from '@rollup/plugin-replace';
+import typescript from '@rollup/plugin-typescript';
+
+import tsConfig from './tsconfig.json';
+
+const browserConfig = {
+  input: 'src/index.ts',
+  output: {
+    dir: 'dist',
+    entryFileNames: 'core.umd.js',
+    exports: 'named',
+    format: 'umd',
```

Comment:
Curious is esm also supported?

Reply:
I need to add support for this later. High priority ATM but not blocking this release.